Cargo.toml

```diff
@@ -28,11 +28,13 @@ futures = "0.1.11"
 futures-cpupool = "0.1"
 hyper = { git = "https://github.com/hyperium/hyper", optional = true }
 hyper-tls = { git = "https://github.com/hyperium/hyper-tls", optional = true }
+jsonwebtoken = { version = "2.0", optional = true }
 libc = "0.2.10"
 local-encoding = "0.2.0"
 log = "0.3.6"
 lru-disk-cache = { path = "lru-disk-cache" }
 number_prefix = "0.2.5"
+openssl = { version = "0.9", optional = true }
```

Comment:
The last commit was two years ago, and there's a note on the repo: "Disclaimer: this is a personal project that may eat your laundry." Converting from PEM to DER seems like a pretty stable operation, but the inactivity of the repo makes me wonder if this is the best idea. What do you think?